packages/client/src/alerts/followCode.vue

```diff
@@ -0,0 +1,137 @@
+<script>
+import * as PIXI from "pixi.js";
+import { gsap } from "gsap";
+import { PixiPlugin } from "gsap/PixiPlugin.js";
+import { RGBSplitFilter, GlitchFilter, DropShadowFilter } from "pixi-filters";
+import { WebfontLoaderPlugin } from "pixi-webfont-loader";
+
+PIXI.Loader.registerPlugin(WebfontLoaderPlugin);
+gsap.registerPlugin(PixiPlugin);
+
+const app = new PIXI.Application({
+  width: 1920,
+  height: 220,
+  backgroundAlpha: 0,
+});
+
+const stage = new PIXI.Container();
+let ticker = PIXI.Ticker.shared;
+let renderer = PIXI.autoDetectRenderer();
+let followText;
+
+export default {
+  name: "alert-follow-code",
+  inject: ["followEvent"],
+  watch: {
+    followEvent() {
+      this.drawFollowEvent();
+    },
+  },
+  methods: {
+    drawFollowEvent() {
+      followText.text = ` followers.add(\"${this.followEvent.user_name.toLowerCase()}\") `;
+      gsap.to(this.$refs.notification, 0.5, {
+        autoAlpha: 1, // appearing from the top
+        repeat: 1, // we repeat only once
+        repeatDelay: 4, // how long it stays on the screen
+        yoyo: true, // yoyo = repeat the animation but reverse
+      });
+    },
+  },
+  async mounted() {
+    gsap.set(this.$refs.notification, { autoAlpha: 0 });
+
+    app.loader.add({ name: "From Google 1", url: "https://fonts.googleapis.com/css2?family=Source+Code+Pro" });
+
+    const pixiLoader = new Promise((resolve) => {
+      app.loader.load(() => resolve());
```

Comment:
The loading could also fail. 
Not sure how you want to handle the error, you could use something like:
```js
app.loader.load((loader, resources) => {
  if(Object.values(resources).some(item => item.error)) reject()
  else resolve()
});
```
It will miss a try/catch mecanism for the await I believe.